lib/flip_card.dart

```diff
@@ -37,7 +37,8 @@ class AnimationCard extends StatelessWidget {
         }
         return Transform(
           transform: transform,
-          alignment: Alignment.center,
+          filterQuality: FilterQuality.none,
```

Comment:
Does this have any performance implications?
Given the [documentation](https://api.flutter.dev/flutter/widgets/Transform/filterQuality.html), it should be faster?